site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -2373,4 +2373,23 @@ private function getStats(Gradeable $gradeable, User $grader, bool $full_stats,
             $total_total += $value * $num_components;
         }
     }
+    
+    /**
+     * @Route("/{_semester}/{_course}/gradeable/{gradeable_id}/feedback/set", methods={"POST"})
```

Comment:
```suggestion
     * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/feedback/set", methods={"POST"})
```

Reply:
Currently, this change breaks the branch: buildCourseUrl in server.js does not include the /courses/.  After testing, before merge, I will make this change.